app/assets/javascripts/tictactoe.js

```diff
@@ -1 +1,139 @@
-// Code your JavaScript / jQuery solution here
+var turn = 0;
+var currentGame = 0;
```

Comment:
Prefer to use `let` or `const` over `var`. See: https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75